app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -377,6 +379,12 @@ public void onCreate(Bundle savedInstanceState) {
 
     initPlayStoreUri();
 
+    if (SDK_INT > VERSION_CODES.LOLLIPOP){
```

Comment:
Please use if ! if you are just going to use one branch. Spacing is still way off from style guidelines.

Reply:
@mhutti1 Fixed it. Does it look correct now?